### PR TITLE
[Doc] Add loads table

### PR DESCRIPTION
**Pull request by amber-create**
Signed-off-by: amber-create <yangyanping@starrocks.com>

Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by EsoragotoSpirit — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by EsoragotoSpirit — DISMISSED** [review]

File: docs/administration/information_schema.md [hunk 1]
```diff
@@ -37,13 +37,44 @@ TABLE_COLLATION: utf8_general_ci
...
+
+### loads
+
+The following fields are provided in `loads`:
+
+| **Field**            | **Description**                                              |
+| -------------------- | ------------------------------------------------------------ |
+| JOB_ID               | The unique ID assigned by StarRocks to identify the load job. |
+| LABEL                | The label of the load job.                                   |
+| DATABASE_NAME        | The name of the database to which the destination StarRocks tabes belong. |
```
Comment (evelynzhaojie) [inline docs/administration/information_schema.md:57]: tabes  --- tables

File: docs/administration/information_schema.md [hunk 2]
```diff
@@ -37,13 +37,44 @@ TABLE_COLLATION: utf8_general_ci
...
+| PROGRESS             | The progress of the ETL stage and LOADING stage of the load job. |
+| TYPE                 | The type of the load job. For Broker Load, the return value is `BROKER`. For INSERT, the return value is `INSERT`. |
+| PRIORITY             | The priority of the load job. Valid values: `HIGHEST`, `HIGH`, `NORMAL`, `LOW`, and `LOWEST`. |
+| SCAN_ROWS            | The number of data rows that are scanned.                    |
+| FILTERED_ROWS        | The number of data rows that are filtered out due to inadequate data quality. |
+| UNSELECTED_ROWS      | The number of data rows that are filtered out due to the conditions specified in the WHERE clause. |
+| SINK_ROWS            | The number of data rows that are loaded.                     |
+| ETL_INFO             | The ETL details of the load job. A non-empty is returned only for Spark Load. For any other types of load jobs, an empty value is returned. |
```
Comment (evelynzhaojie) [inline docs/administration/information_schema.md:66]: A non-empty --- maybe the noun is missing ?

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by EsoragotoSpirit — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Comment by amber-create** [comment]
@mergify backport branch-3.1

**Pull request merged**